recipes/autocfg/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "autocfg" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/autocfg-{{ version }}.tar.gz
+  sha256: 749986b4f3b3bd85b15298734bf8fa4a590e6c34a314ac515025e058ed76c319
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pyyaml
+    - dataclasses  # [py<37]
+
+test:
+  imports:
+    - autocfg
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/zhreshold/autocfg
+  summary: Deep learning configuration
+  license: Apache-2.0
+  license_file: LICENSE
```

Comment:
The copy of this file can be removed here.

By referencing the file name/location under `license_file`, conda will automatically include a copy from the source at build time.